tests/e2e/util.go

```diff
@@ -6138,3 +6153,22 @@ func readDataFromRawBlockVolume(ns string, podName string, devicePath string, te
 	framework.ExpectNoError(err, fmt.Sprintf("failed to read device: %q inside the pod: %q", devicePath, podName))
 	gomega.Expect(strings.Contains(output, testData)).NotTo(gomega.BeFalse())
 }
+
+// getBlockDevSizeInSectors returns size of block device at given path
+func getBlockDevSizeInSectors(f *framework.Framework, ns string, pod *v1.Pod, devicePath string) (int64, error) {
+	cmd := []string{"exec", pod.Name, "--namespace=" + ns, "--", "/bin/sh", "-c",
+		fmt.Sprintf("/bin/ls %v", devicePath)}
+	_, err := framework.RunKubectl(ns, cmd...)
+	if err != nil {
+		return -1, fmt.Errorf("failed to get raw device path %v inside pod", devicePath)
+	}
+
+	cmd = []string{"exec", pod.Name, "--namespace=" + ns, "--", "/bin/sh", "-c",
```

Comment:
you can get size in bytes using `blockdev --getsize64` and it can be converted to mb easily, any reason why we are using sectors here

Reply:
No specific reason but this function is used for volume size comparison, so IMO size in MB/sectors should not impact, when both sizes compared are taken in same unit.